### DISCUSSION TITLE
Just bought premium feature - horizontal scrolling big pain

### DISCUSSION THREAD

**Discussion by furkan-c** [AppGround-io/bluetooth-keyboard-and-mouse-support#147] [Bugs & Problems]
The mouse is fantastic. However, can we have dedicated buttons for vertical and horizontal scrolling. Horizontal scrolling is too buggy. The screen just scrolls both horizontally and vertically everytime I try a horizontal scroll. Using a T95Q android box. Please let me know if this cannot be fixed so that I can return the box.

**Comment by ricardohnn** [comment]
I agree, horizontal scroll also doesn't work on ipads
